pilot/pkg/serviceregistry/serviceentry/store.go

```diff
@@ -30,7 +31,7 @@ type hostPort struct {
 type serviceInstancesStore struct {
 	ip2instance map[string][]*model.ServiceInstance
 	// service instances by hostname -> config
-	instances map[instancesKey]map[configKey][]*model.ServiceInstance
+	instances map[instancesKey]map[configKeyWithParent][]*model.ServiceInstance
```

Comment:
can we use map[instancesKey]map[parentKey]map[configKey][]*model.ServiceInstance

Reply:
This is just a hierachy choice.  One maybe clear, one maybe simple to manage. I donot feel strongly.